apps/music/test/marionette/metadata_test.js

```diff
@@ -0,0 +1,73 @@
+/* global require, marionette, setup, suite, test, __dirname */
+'use strict';
+
+var assert = require('assert');
+var Music = require('./lib/music.js');
+var FakeRingtones = require('./lib/fakeringtones.js');
```

Comment:
I've got no idea what the fakeringtones is for, but have you checked whether it is really necessary here?
Since it seems so unrelated to the point of this test, if you really do need it, maybe add a comment just saying that it is necessary but irrelevant.